src/main/java/org/folio/service/impl/AllowedServicePointsForItemLevelRequestService.java

```diff
@@ -41,15 +41,15 @@ protected Collection<String> getLendingTenants(AllowedServicePointsRequest reque
   }
 
   @Override
-  protected AllowedServicePointsResponse getAllowedServicePointsFromLendingTenant(
+  protected AllowedServicePointsResponse getAllowedServicePointsFromTenant(
     AllowedServicePointsRequest request, String patronGroupId, String tenantId) {
 
     log.info("getAllowedServicePointsFromLendingTenant:: parameters: request: {}, " +
```

Comment:
You've changed the name of the method, but still logging the old name

Reply:
fixed